File: apps/mobile/targets/positions-widget/Sources/Rendering/Common/Stats/TokenStatHelper.swift
import Foundation

enum TokenStatHelper {
  static func feeString(for symbol: String, in fees: [WidgetToken]) -> String? {
    guard let token = fees.first(where: { $0.symbol == symbol }) else { return nil }
    guard token.formatted != "0" else { return nil }
    return "+\(token.formatted)"
  }
}

File: apps/mobile/modules/widget-bridge/android/src/main/java/expo/modules/widgetbridge/widget/util/TokenStatHelper.kt
package expo.modules.widgetbridge.widget.util

import expo.modules.widgetbridge.widget.data.WidgetToken

object TokenStatHelper {
  fun feeString(symbol: String?, fees: List<WidgetToken>): String? {
    if (symbol == null) return null
    val token = fees.firstOrNull { it.symbol == symbol } ?: return null
    if (token.formatted == "0") return null
    return "+${token.formatted}"
  }
}
